src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -122,5 +123,51 @@ public void AnalyzeDirectory_DirectoryWithOtherJsonFiles_LogsExpectedErrorInSari
             Assert.AreNotEqual(null, toolNotifications[1]["exception"]);
             Assert.AreEqual(null, toolNotifications[2]["exception"]);
         }
+
+        [TestMethod]
+        public void AnalyzeDirectory_ExecutionWithErrorAndWarningAndNoVerboseMode_PrintsExpectedLogSummary()
+        {
+            var directoryToAnalyze = GetFilePath("ToTestSummaryLogger");
+
+            var expectedLogSummary = "2 error(s) and 1 warning(s) were found during the execution, please refer to the original messages above" +
+                $"{Environment.NewLine}The verbose mode (option -v or --verbose) can be used to obtain even more information about the execution" +
+                $"{Environment.NewLine}Summary of the errors:" +
+                $"{Environment.NewLine}\t1 instance(s) of: An exception occurred while analyzing a template" +
+                $"{Environment.NewLine}\t1 instance(s) of: Unable to analyze 1 file(s): {Path.Combine(directoryToAnalyze, "ReportsError.json")}" +
```

Comment:
Is the one template resulting in two errors logged?  Ideally a failed template should only be logged once.

Reply:
Yes, one error is logged in AnalyzeTemplate, and the second one after the execution returns to AnalyzeDirectory. Should I add an issue? Fixing this properly to me seems out of scope for this PR

Reply:
Oh, I see, the second error is the directory summary.  That's fine, thanks for clarifying.